docs/intro/tutorial.rst

```diff
@@ -78,8 +79,8 @@ Our first Spider
 
 Spiders are classes that you define and that Scrapy uses to scrape information
 from a website (or a group of websites). They must subclass
-:class:`~scrapy.Spider` and define the initial requests to make,
-optionally how to follow links in the pages, and how to parse the downloaded
+:class:`~scrapy.Spider` and define the initial requests to be made.
```

Comment:
This seems arbitrary to me.

Reply:
Would this be better?:

They must subclass :class:`~scrapy.Spider` and define the initial requests to be made, and optionally, how to follow links in the pages and parse the downloaded page content to extract data.

Reply:
Sounds good, but isn’t the “the” in “the pages” unnecessary?

Reply:
Yes you're right the "the" is unnecessary